main.tf

```diff
@@ -201,9 +212,11 @@ resource "google_cloud_run_v2_service" "creative_studio" {
         }
       }
     }
-    service_account = google_service_account.creative_studio.email
+    max_instance_request_concurrency = 4
```

Comment:
Is there a reason to limit the request concurrency to 4? [Default is 80](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#max_instance_request_concurrency-1) given that even originally 1 full CPU was provisioned.